src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,361 @@
+import React, { Component } from "react";
+import PropType from "prop-types";
+
+import { FaBars, FaSearch } from "react-icons/fa";
+
+import BillFilter from "../BillFilter";
+import SimpleFilter from "../SimpleFilter";
+
+import "./styles.scss";
+import { Button } from "shards-react";
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentSorting: "Newest",      
+      dateFilters: {
+        startDate: { selected: false, value: "" },
+        endDate: { selected: false, value: "" },
+      },
+      statusFilter: { resolved: false, open: false, in_progess: false },
+      toggle: {
+        short: false,
+        long: false,
+      },
+      filterToggles: {
+        statusOpened: false,
+        categoryOpened: false,
+        dateOpened: false,
+      },
+
+      // TODO
```

Comment:
If you really have to keep a TODO within the code base, make a task and put the task number here